src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,36 @@
+.container {
+  @include center-flex;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  @include center-flex;
```

Comment:
your center-flex contains margins, but it's not allowed to set margins for your BEM block